docs/helpers/CameraHelper.md

```diff
@@ -0,0 +1,90 @@
+---
+title: CameraHelper
+author: skommireddi
+description: The CameraHelper provides helper methods to easily use the available camera frame sources to preview video, capture video frames and software bitmaps.
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, CameraHelper, Camera, Frame Source, Video Frame, Software Bitmap
+---
+
+# CameraHelper
+
+The **CameraHelper** provides helper methods tto easily use the available camera frame sources to preview video, capture video frames and software bitmaps. The helper currently shows camera frame sources that support color video preview or video record streams. 
+
+> [!IMPORTANT] Make sure you have the webcam capability enabled for your app to access the device's camera.
+
+## Syntax
+
+```csharp
+// Creates a Camera Helper and gets video frames from an available frame source.
+using Microsoft.Toolkit.Uwp.Helpers.CameraHelper;
+
+CameraHelper cameraHelper = new CameraHelper();
+var result = await _cameraHelper.InitializeAndStartCaptureAsync();
+
+// Camera Initialization and Capture failed for some reason
+if(result != CameraHelperResult.Success)
+{
+// get error information
+var errorMessage = result.ToString();
+}
+
+// Subscribe to get frames as they arrive
+cameraHelper.FrameArrived += CameraHelper_FrameArrived;
+
+private void CameraHelper_FrameArrived(object sender, FrameEventArgs e)
+{
+  // Gets the current video frame
+  VideoFrame currentVideoFrame  = e.VideoFrame;
+
+  // Gets the software bitmap image
+  SoftwareBitmap softwareBitmap = e.SoftwareBitmap;
+}
+```
+
+## Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| FrameSource| MediaFrameSource| Gets the currently selected camera MediaFrameSource|
+| FrameSourceGroups| IReadOnlyList<MediaFrameSourceGroup>| Gets a read only list of MediaFrameSourceGroups that support color video record or video preview streams.|
+
+## Methods
+
+| Methods | Return Type | Description |
+| -- | -- | -- |
+| InitializeAndStartCaptureAsync(MediaFrameSourceGroup group = null) | Task<CameraHelperResult>| Initializes Camera Media Capture settings and initializes Frame Reader to capture frames in real time. If no MediaFrameSourceGroup is provided, it selects the first available camera source to  use for media capture. 
+| Dispose() | void | Use this method to dispose resources |
+
+## Events
+
+| Events | Description |
+| -- | -- |
+| FrameArrived| Fires when a new frame arrives.|
+
+```csharp
+private void CameraHelper_FrameArrived(object sender, FrameEventArgs e)
+{
+  // Gets the current video frame
+  VideoFrame currentVideoFrame  = e.VideoFrame;
+
+  // Gets the software bitmap image
+  SoftwareBitmap softwareBitmap = e.SoftwareBitmap;
+}
+```
+
+## Sample Code
+
+[CameraHelper Sample Page]
+((https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/CameraHelper))
```

Comment:
Change this to match [documentation template](https://github.com/Microsoft/UWPCommunityToolkit/blob/master/docs/.template.md)

[CameraHelper Sample Page Source](https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/CameraHelper). You can see this in action in [UWP Community Toolkit Sample App](https://www.microsoft.com/store/apps/9NBLGGH4TLCQ).